app/controllers/works_controller.rb

```diff
@@ -114,8 +114,6 @@ def update
     if current_user.blank? || !@work.editable_by?(current_user)
       Rails.logger.warn("Unauthorized attempt to update work #{@work.id} by user #{current_user.uid}")
       redirect_to root_path
-    elsif @work.approved? && @work.submitted_by?(current_user)
```

Comment:
The requirement that a user can not edit an item they deposited was ticketed here: https://github.com/pulibrary/pdc_describe/issues/255.  Does removing this code introduce a bug?